OpenDreamRuntime/Procs/Native/DreamProcNativeWorld.cs

```diff
@@ -38,5 +38,67 @@ public static async Task<DreamValue> NativeProc_Export(AsyncNativeProc.State sta
 
             return new DreamValue(list);
         }
+
+        [DreamProc("GetConfig")]
+        [DreamProcParameter("config_set", Type = DreamValue.DreamValueType.String)]
+        [DreamProcParameter("param", Type = DreamValue.DreamValueType.String, DefaultValue = null)]
+        public static DreamValue NativeProc_GetConfig(DreamObject src, DreamObject usr, DreamProcArguments arguments)
+        {
+            var config_set = arguments.GetArgument(0, "config_set").Stringify();
```

Comment:
`Stringify()` is used to convert any type of DreamValue into a string, often to be displayed to a user (like in `world.log << ...`). Doing it this way would let me do something like this (an error in byond):
```
var/obj/O = new()
O.name = "env"
world.log << world.GetConfig(O, "ENV_VAR")
```

What you want instead is `TryGetValueAsString()` and to handle cases where you weren't passed a string. In this case it should error, as that's what byond does here.